server/controllers/AuthController.js

```diff
@@ -1,93 +1,81 @@
-const User = require('../models/User')
-const bcrypt = require('bcrypt')
-const jwt = require('jsonwebtoken')
+const User = require('../models/User');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
 
 // Registration process
 const register = (req, res, next) => {
-    var userName = req.body.userName
+    var userName = req.body.userName;
 
-    User.findOne({userName: userName})
-    .then(user => {
-        if(user)
-        {
-            res.json({
-                message: "User already exists!"
-            })
-        } else {
-            bcrypt.hash(req.body.password, 10, function(err, hashedPass){
-                if(err)
-                {
-                    res.json({
-                        error: err
-                    })
-                }
-
-                let user = new User({
-                    userName: req.body.userName,
-                    password: hashedPass
-                })
-            
-                user.save()
-                .then(user =>{
-                    res.json({
-                        message: 'User Added Successfully!'
-                    })
-                })
-            
-                .catch(error => {
-                    res.json({
-                        message: 'An error occured!'
-                    })
-                })
-            })
-        }
-    });
-
-    
+    User.findOne({ userName: userName })
+        .then(user => {
+            if (user) {
+                return res.json({
```

Comment:
```suggestion
                return res.status(409).json({
```

Instead of returning 200 every time, differentiate the value by returning a proper http status